README.md

```diff
@@ -140,7 +140,7 @@ I would like to thank all Micronauts for their support
 
 > Add a link to your deployed project.
 
-- [My_Portfolio](https://kwaku28.github.io/My-Portfolio/)
+- [My_Portfolio](https://mike47ip.github.io/mobilePortfolioWeb/)
 

```

Comment:
- [ ] Good job by adding a working link to your deployed projectf. However, I noticed that some images are broken to your deployed page, this is because of the wrong path of your image(s) which contains some typo errors. You should write `<img src="./images/Disabled.png" alt="scroll" />` instead of `<img src="./Images/Disabled.png" alt="scroll" />`, your `images` folder is not capitalized.

Reply:
> * [x]  Good job by adding a working link to your deployed projectf. However, I noticed that some images are broken to your deployed page, this is because of the wrong path of your image(s) which contains some typo errors. You should write `<img src="./images/Disabled.png" alt="scroll" />` instead of `<img src="./Images/Disabled.png" alt="scroll" />`, your `images` folder is not capitalized.




I changed the file name of my Images folder to capital "Images" on the local computer but it didn't appear here. You cannot change the name of the folder remotely on github either. It did not work anyway. I also made the Images in my style.css url directory capital but same thing. 

Please overlook this technicalities and concentrate on the real task. Thank you.